### PR TITLE
fix: try send/sendNative with callStatic

### PR DESCRIPTION
**Pull request by ErinHales**
Test if a transaction with succeed/fail before trying to send.

## Motivation

Don't spend gas if we know it's going to fail

## Solution

Implement `callStatic` call

## PR Checklist

- [x] Added Tests (tested locally)
- [x] Updated Documentation
- [x] Updated CHANGELOG.md for the appropriate package

### CODE REVIEW

File: packages/sdk-bridge/src/BridgeContext.ts [hunk 1]
```diff
@@ -279,17 +279,27 @@ export class BridgeContext extends NomadContext {
       await tx.wait();
     }
 
-    const tx = await fromBridge.bridgeRouter.populateTransaction.send(
+    if (!overrides.gasLimit) {
+      overrides.gasLimit = BigNumber.from(350000)
```
Comment (prestwich) [inline packages/sdk-bridge/src/BridgeContext.ts:283]: magic number to named constant?

**Review by anna-carroll — APPROVED** [review]
very nice :,)

File: packages/sdk-bridge/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
 ### 1.0.0-rc.19
 
 - fix: numeric fault overflow when supporting infinite token approval
+- feat: try send/sendNative and process calls using callStatic
```
Comment (prestwich) [inline packages/sdk-bridge/CHANGELOG.md:8]: shouldn't this be in unreleased?

**Review by prestwich — CHANGES_REQUESTED** [review]

File: packages/sdk-bridge/src/BridgeContext.ts [hunk 3]
```diff
@@ -279,17 +279,27 @@ export class BridgeContext extends NomadContext {
       await tx.wait();
     }
 
-    const tx = await fromBridge.bridgeRouter.populateTransaction.send(
+    if (!overrides.gasLimit) {
+      overrides.gasLimit = BigNumber.from(350000);
```
[the remark this answers is not in the document]
Comment (ErinHales) [reply packages/sdk-bridge/src/BridgeContext.ts:283]: What do you mean by magic number?

You just want it like this?
```
const gasLimit = BigNumber.from(350000);
overrides.gasLimit = gasLimit
```

File: packages/sdk-bridge/CHANGELOG.md (hunk 2 above)
Comment (ErinHales) [reply packages/sdk-bridge/CHANGELOG.md:8]: That package was about to be released but I asked to get this in at the same time

**Review by prestwich — APPROVED** [review]

**Pull request merged**